### ISSUE TITLE
`npx remark` stopped working

### ISSUE THREAD

**Issue by Xunnamius** [remarkjs/remark#1325]
As of ~4 days ago, invoking remark via `npx remark` in my pipelines seems to have stopped working. Given the timing, it may have to do with the latest releases of the remark and remark-cli packages.

### Initial checklist

- [X] I read the [support docs](https://github.com/remarkjs/.github/blob/main/support.md)
- [X] I read the [contributing guide](https://github.com/remarkjs/.github/blob/main/contributing.md)
- [X] I agree to follow the [code of conduct](https://github.com/remarkjs/.github/blob/main/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Aremarkjs&type=Issues) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

remark: 15.0.1, remark-cli: 12.0.1

### Steps to reproduce

1. Run `npx remark --help` in a terminal with `npx` in the path.

### Expected behavior

Should run remark CLI like normal.

### Actual behavior

```
npm ERR! could not determine executable to run
```

### Runtime

Node v21.7.3

### Package manager

NPM 10.5.0

### OS

Linux (Ubuntu)

### Build and bundle tools

N/A

**Comment by wooorm** [comment]
I can’t imagine something happening 4 days ago? There was a CLI release 7 days ago?
This is also around the time of Node 22 coming out, with a new npm probably?

**Comment by wooorm** [comment]
Wait, has this ever worked? Because you always had to *install* `remark-cli`, and then *use* it as the command `remark`?

```sh
npx --package remark-cli -- remark --help
```

Perhaps `npx` worked before when you already had `remark-cli` installed locally, and thus had the `remark` command?

**Comment by Xunnamius** [comment]
Looking at a handful of my other remark-powered projects, they do have remark-cli installed locally as you describe, and this newer project does not.

The strange thing is, everything was working fine until I updated deps last Friday. Perhaps remark-cli was a transitive dep of some deeper package that dropped it after I updated, which would explain why this didn't break here earlier. Either way, installing remark-cli (or just doing `npx remark-cli`) returned everything to working order, so this likely isn't your problem and we can close this issue.

Thanks for the quick reply and apologies for the time sink!
